### DISCUSSION TITLE
Tackling filesystem issues using the REPL Erase Method

### DISCUSSION THREAD

**Discussion by chadoulis** [STELLA-Landsat/STELLA#26] [Build Notes: How do I make a STELLA?]
I was facing the following issue: The code.py file was corrupt, appearing to have a size of 0KB, was not recognized by the Mu Editor, and I could not delete it or replace it. The solution that worked for me (and may be useful for similar filesystem issues) was the one initially described in the [AdaFruit CircuitPython Documentation](https://docs.circuitpython.org/en/latest/docs/troubleshooting.html). In particular, I opened the Mu Editor, connected to the CircuitPython REPL by pressing any key and did the following:

```
Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.

Press any key to enter the REPL. Use CTRL-D to reload.

Adafruit CircuitPython 8.1.0 on 2023-05-22; Adafruit Feather nRF52840 Express with nRF52840
>>> import storage
>>> storage.erase_filesystem()
```

This resolved my issue. I created this thread so anyone facing something similar can try this approach.

**Comment by paulmirel** [comment]
Thanks for posting this. I suspect the filesystem go scrambled with the intermittent power the microcontroller was getting from the incomplete solder joints. I have not personally encountered corrupt code.py files.
